Enhancer/EnhancerInterface.php

```diff
@@ -28,8 +27,8 @@
      *
      *     $context->addData('foobar', 'Some value');
      *
-     * @param Context Serialization context
-     * @param resource The resource being serialized
+     * @param []       $data        Context Serialization context
+     * @param Resource $resource    The resource being serialized
      */
-    public function enhance(array $data, PuliResource $resource);
+    public function enhance(array $data, Resource $resource);
```

Comment:
I had to explicitly include Puli `@beta` in ResourceBundle:

    https://github.com/symfony-cmf/resource-bundle/pull/31/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R17

On Fri, Jun 03, 2016 at 06:38:48AM -0700, Maximilian Berghoff wrote:
>    In [1]Enhancer/EnhancerInterface.php:
> 
>  >       */
>  > -    public function enhance(array $data, PuliResource $resource);
>  > +    public function enhance(array $data, Resource $resource);
> 
>    Now I am completely confused. Localy i still got Resource. Your Link sais
>    the change was the other way around, and i trust you, so i have the wrong
>    version and i think i did the same wrong change on resource-bundle too. No
>    clue if [2]@dantleech took that error too. But at least we should try to
>    force to get the latest version of that bundle.
> 
>    —
>    You are receiving this because you were mentioned.
>    Reply to this email directly, [3]view it on GitHub, or [4]mute the thread.
> 
>    Reverse link: [5]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/symfony-cmf/resource-rest-bundle/pull/31#discussion_r65707989
>    2. https://github.com/dantleech
>    3. https://github.com/symfony-cmf/resource-rest-bundle/pull/31/files/fcd3f0aae763a6b30211a5e3c1e51fc71508a7c9#r65707989
>    4. https://github.com/notifications/unsubscribe/AAgZcXom6yfXsS9L4BesP9QqZ8tUMXAcks5qIC5ogaJpZM4Ik8Qk
>    5. https://github.com/symfony-cmf/resource-rest-bundle/pull/31/files/fcd3f0aae763a6b30211a5e3c1e51fc71508a7c9#r65707989